### ISSUE TITLE
Close button should have meaningful text

### ISSUE THREAD

**Issue by smashercosmo** [fkhadra/react-toastify#468]
**Do you want to request a *feature* or report a *bug*?**
Regression

**What is the current behavior?**
Close button doesn't have any meaningful text. It used to have ```aria-label=close```

**What is the expected behavior?**
Close button should have meaningful text either via button contents or via aria-label attribute

**Comment by fkhadra** [comment]
@smashercosmo fix published. I've also added a test for this. Thanks for reporting the issue
